ExampleMod/Common/GlobalItems/GelGlobalItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections.Generic;
+using Terraria;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Common.GlobalItems
+{
+	// This is another part of the ExampleShiftClickSlotPlayer.cs that adds a tooltip line to the gel
+	public class GelGlobalItem : GlobalItem
+	{
+		public override void ModifyTooltips(Item item, List<TooltipLine> tooltips) {
```

Comment:
You're missing an override of `AppliesToEntity` so this just added your tooltip to every item in the game

Reply:
Forgot to do so. Now it's fixed